x/wasm/types/errors.go

```diff
@@ -85,6 +85,7 @@ var (
 	ErrNoSuchCodeFn = WasmVMFlavouredErrorFactory(sdkErrors.Register(DefaultCodespace, 28, "no such code"),
 		func(id uint64) error { return wasmvmtypes.NoSuchCode{CodeID: id} },
 	)
+	ErrZero = sdkErrors.Register(DefaultCodespace, 29, "0 is not accepted")
```

Comment:
We moved away from more concrete error types. How about using `ErrInvalid` instead?